### DISCUSSION TITLE
第3节课笔记(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#176] [笔记]
希望大家坚持下去✊ 坚持肯定会有收获的！🌹

**Comment by zhanghui-china** [comment]
[https://zhuanlan.zhihu.com/p/676720943](url)

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/676296328

**Comment by qianlian-mozi** [comment]
https://blog.csdn.net/m0_54824511/article/details/135491969?spm=1001.2014.3001.5501

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%B8%89%E8%8A%82.md

**Comment by dukai289** [comment]
https://github.com/dukai289/Learning/blob/master/Learning_InternLM/3_RAG.md

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/note/3/%E7%AC%AC%E4%B8%89%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135497589

**Comment by hellokid123** [comment]
https://github.com/hellokid123/internllm-notes/blob/main/notes/%E7%AC%AC3%E8%AF%BE.md

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/3%E3%80%81%E5%9F%BA%E4%BA%8E-InternLM-%E5%92%8C-LangChain-%E6%90%AD%E5%BB%BA%E7%9F%A5%E8%AF%86%E5%BA%93

**Comment by LichKing-W** [comment]
https://zhuanlan.zhihu.com/p/676992236

**Comment by codefd** [comment]
https://blog.csdn.net/googteo/article/details/135504622?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135504622%22%2C%22source%22%3A%22googteo%22%7D

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135467720?spm=1001.2014.3001.5502

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/4202b3b2-3ded-4686-a745-03eb42f909cd
【FlowUs 息流】基于 InternLM 和 LangChain 搭建你的知识库

**Comment by Fulin-Guo** [comment]
https://github.com/wdnmdm/Team_learning/blob/main/InternLM/Chapter3.md

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/3.md

**Comment by Lyusijie** [comment]
https://blog.csdn.net/weixin_44986362/article/details/135493480?spm=1001.2014.3001.5501

**Comment by godlikeS97** [comment]
https://blog.csdn.net/godlikesun97/article/details/135535723?spm=1001.2014.3001.5501

**Comment by Kongsea** [comment]
https://blog.csdn.net/deeepsea/article/details/135537313

**Comment by Unicornlyy** [comment]
https://blog.csdn.net/m0_68165821/article/details/135551666

**Comment by Vacili-f** [comment]
https://github.com/Vacili-f/InternLM/blob/main/%E7%AC%94%E8%AE%B0/%E7%AC%AC%E4%B8%89%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md
